samples/test/rollout.js

```diff
@@ -0,0 +1,207 @@
+    const [operation] = await deploymentClient.createGameServerDeployment(
+      request
+    );
+    await operation.promise();
+
+    const request2 = {
+      parent: configClient.gameServerDeploymentPath(
+        projectId,
+        'global',
+        deploymentId
+      ),
+      configId: configId,
+    };
+    const [operation2] = await configClient.createGameServerConfig(request2);
+    await operation2.promise();
+
+    const request3 = {
+      parent: `projects/${projectId}/locations/${realmLocation}`,
+      realmId: realmId,
+      realm: {
+        timeZone: 'US/Pacific',
+        description: 'Test Game Server realm',
+      },
+    };
+
+    const [operation3] = await realmClient.createRealm(request3);
+    await operation3.promise();
+  });
+
+  it('should update rollout default', async () => {
+    const update_default_output = execSync(
+      `node update_rollout_default.js ${projectId} ${deploymentId} ${configId}`
+    );
+
+    assert.match(update_default_output, /Deployment updated:/);
+
+    const get_rollout_output = execSync(
+      `node get_rollout.js ${projectId} ${deploymentId} ${configId}`
+    );
+
+    const configName = configClient.gameServerConfigPath(
+      projectId,
+      'global',
+      deploymentId,
+      configId
+    );
+    assert.match(get_rollout_output, /Rollout name:/);
+    assert.match(
+      get_rollout_output,
+      new RegExp(`Rollout default: ${configName}`)
+    );
+  });
+
+  it('should remove rollout default', async () => {
+    const remove_default_output = execSync(
+      `node update_rollout_remove_default.js ${projectId} ${deploymentId} ${configId}`
+    );
+
+    assert.match(remove_default_output, /Deployment updated:/);
+
+    const request = {
+      // The full resource name
+      name: deploymentClient.gameServerDeploymentPath(
+        projectId,
+        'global',
+        deploymentId
+      ),
+    };
+
+    const [rollout] = await deploymentClient.getGameServerDeploymentRollout(
+      request
+    );
+    assert(rollout.defaultGameServerConfig === '');
+  });
+
+  it('should update rollout override', async () => {
+    const update_override_output = execSync(
+      `node update_rollout_override.js ${projectId} ${deploymentId} ${configId} ${realmId} ${realmLocation}`
+    );
+    assert.match(update_override_output, /Deployment updated:/);
+
+    const get_rollout_output = execSync(
+      `node get_rollout.js ${projectId} ${deploymentId} ${configId}`
+    );
+
+    const configName = configClient.gameServerConfigPath(
+      projectId,
+      'global',
+      deploymentId,
+      configId
+    );
+    const realmName = realmClient.realmPath(projectId, realmLocation, realmId);
+    assert.match(
+      get_rollout_output,
+      new RegExp(`Rollout config overrides: .*${configName}`)
+    );
+    assert.match(get_rollout_output, new RegExp(`${realmName}`));
+  });
+
+  it('should remove rollout override', async () => {
+    const remove_override_output = execSync(
```

Comment:
Here (and elsewhere) - snake case is a very python thing 🐍  In node.js, we use camelCase.